### PR TITLE
Explicit linkages for ODCID/OCID TP

### LINKED ISSUE #2926: ICID or OCID?
End of section 18, transport

> A client MUST NOT include an original connection ID, a stateless reset token, or a preferred address. A server MUST treat receipt of any of these transport parameters as a connection error of type TRANSPORT_PARAMETER_ERROR.

It seems the term is "initial connection ID" not "original connection ID", at least I can't find it defined anywhere in transport.

### PR DESCRIPTION
**Pull request by MikeBishop**
Fixes #2926.

### CODE REVIEW

**Comment by MikeBishop** [comment]
Note that #3120 may want to change the phrasing of the ODCID field, depending on which PR lands first.

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -4412,9 +4412,10 @@ The following transport parameters are defined:
 original_connection_id (0x0000):
 
 : The value of the Destination Connection ID field from the first Initial packet
-  sent by the client.  This transport parameter is only sent by a server.  A
-  server MUST include the original_connection_id transport parameter if it sent
-  a Retry packet.
+  sent by the client.  This transport parameter is only sent by a server.  This
+  is the same value sent in the "Original Destination Connection ID" field of a
+  Retry packet (see {{packet-retry}}).  A server MUST include the
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md]: Note that David's proposal for integrity protection of the Retry packet changes this text to point to the original Initial sent by the client.  You should probably do the same.

Comment (MikeBishop) [reply draft-ietf-quic-transport.md]: Can you point to the specific text you're looking at?

David's text in #3120:
> value of the Destination Connection ID from the Initial packet that this Retry is in response to.

Text here:
> value of the Destination Connection ID field from the first Initial packet sent by the client

These seem to indicate the same Initial packet.

Comment (martinthomson) [reply draft-ietf-quic-transport.md]: I might have misremembered, sorry.  In #3120, the text I was concerned about was

> If the server sends a Retry packet, it MUST include the Destination Connection ID field from the client's first Initial packet in the transport parameter.

But you didn't touch that.

**Pull request merged**